src/persist/src/indexed/mod.rs

```diff
@@ -1203,7 +1248,7 @@ mod tests {
 
         // Empty things are empty.
         let ArrangementSnapshot(unsealed, trace, seqno, seal_frontier) =
-            block_on(|res| i.snapshot(id, res))?;
+            block_on(|res| i.apply(Cmd::CmdRead(CmdRead::Snapshot(id, res))))?;
```

Comment:
these are all clunky but straightforward enough that it seemed okay to be. the only one I made a test wrapper for with the old signature was Register because it has to turn a bunch of things into owned strings, but curious if you think even that is worth it

Reply:
I think these are fine it seems verbose but easily fits in one line and is straightforward like you said. I don't have strong feelings on the register bit -- I would maybe advocate for a higher level test helper like one that creates an index, registers a stream named "0", and fills it with some starting data, and returns the index and read/write handles or something? I feel like a lot of the tests do at least the first 2 things, and some do all 3?